recipes/pytensor/meta.yaml

```
@@ -0,0 +1,110 @@
+{% set version = "2.8.10" %}
+
+package:
+  # https://github.com/conda/conda-build/issues/3933
+  name: pytensor-suite 
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/pytensor/pytensor-{{ version }}.tar.gz
+  sha256: 1cc26ebf6a5a8678b37bcb6ae497334bca3e3f7e27cb7815b0c9af0c0deb83f3
+  patches:
+
+build:
+  number: 0
+
+outputs:
+  - name: pytensor-base
+    build:
+      script:
+        - python -m pip install . --no-deps -vv
+      entry_points:
+        - pytensor-cache = bin.pytensor_cache:main
+    requirements:
+      build:
+        - python                                 # [build_platform != target_platform]
+        - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+      host:
+        - pip
+        - python
+      run:
+        - python
+
+        # Ref: <https://github.com/pymc-devs/pytensor/blob/rel-{{ version }}/setup.py>
+        - numpy >=1.17.0
+        - scipy >=0.14
+        - filelock
+        - etuples
+        - logical-unification
+        - miniKanren
+        - cons
+        - typing_extensions
+        - setuptools >=48.0.0
+    test:
+      imports:
+        - pytensor
+      commands:
+        - pytensor-cache help
+        - pip check
+        - python check-for-warnings.py allowed-warnings-base.txt
+      requires:
+        - pip
+      files:
+        - check-for-warnings.py
+        - allowed-warnings-base.txt
+  - name: pytensor
```

Not important but a space here help us find sub-packages.

```suggestion

  - name: pytensor
```